prime-router/settings/STLTs/MO/mo-phd.yml

```diff
@@ -33,8 +33,7 @@
         replaceUnicodeWithAscii: false
         useBlankInsteadOfUnknown: null
         truncateHDNamespaceIds: true
-        truncateHl7Fields: "MSH-3-1, PID-3-3, PID-3-6, ORC-2-2, ORC-7-10-5, OBR-2-2,\
-        \ OBR-3-2, SPM-2-1-2, SPM-2-2-2, ORC-21-1, OBX-23-1, BX-15.1, OBX-23.10 "
+        truncateHl7Fields: "MSH-3-1, PID-3-3, PID-3-6, ORC-2-2, ORC-7-10-5, OBR-2-2, OBR-3-2, SPM-2-1-2, SPM-2-2-2, ORC-21-1, OBX-23-1, BX-15.1, OBX-23.10 "
```

Comment:
When I was testing I found: 

- `PID-3-6` only worked if you specified it as `PID-3-6-1`
- `OBX-23.10` works as `OBX-23-10`

and `BX-15.1 `is probably a typo but it would use the "-" notation too

Reply:
Good catches, thank you!!!